Libraries/LibGfx/Forward.h

```diff
@@ -46,7 +46,7 @@ class Point;
 class Rect;
 class Size;
 class StylePainter;
-class SystemTheme;
+struct SystemTheme;
```

Comment:
> SystemTheme.h:81:1: error: 'SystemTheme' defined as a struct here but previously declared as a class; this is valid, but may result in linker errors under the Microsoft C++ ABI

Yeah I guess we shouldn't risk getting linker errors under the Microsoft C++ ABI :smiley:

Reply:
Yeah I wouldn't care either but it blocks compilation with `-Werror` on.

Reply:
No, I get it, just found it somewhat funny 😀